source/documentation/deploying-an-app/use-circleci-to-upgrade-app.html.md.erb

```diff
@@ -53,11 +117,37 @@ A Service Account is created in the [namespace creation github repository](https
   token: ZX...EE=
 ```
 
+There will be a secret for the circle ci service account. The secret contains the ca.crt and token which we will be injected into the circle CI image container as environment variables. This will allow it to authenticate against the cluster. 
```

Comment:
typo: "which we will be injected" -> "which we will inject" or "which will be injected"

Although, thinking about it, "which we will make available to the circle CI image container..." might be easier to understand.

Reply:
done.